Formula/latino.rb

```diff
@@ -0,0 +1,33 @@
+class Latino < Formula
+  desc "Lenguaje de programación de código abierto para latinos y de habla hispana"
+  homepage "https://www.lenguajelatino.org/"
+  license "MIT"
+  stable do
+    url "https://github.com/lenguaje-latino/Latino/archive/v1.3.0.tar.gz"
+    sha256 "28118a047265394ad9a5fcb2c76895d08af4b43a5886f52afce98a0f8cdc86fe"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  head do
+    url "https://github.com/lenguaje-latino/Latino.git"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  fails_with :clang do
+    build 600
+    cause "multiple configure and compile errors"
+  end
+  fails_with :gcc do
+    build 600
+    cause "multiple configure and compile errors"
+  end
+  def install
+    mv "logo/Latino-logo.png", "logo/desktop.png"
```

Comment:
Why is this needed?

Reply:
the tagged/stable version calls the logo `Latino-logo.png` but in the cmake.txt it references `desktop.png` which causes an error with the build. Simply re-naming it allows the build to succeed. This will not be needed once the next stable version comes out, whenever that might be